webapp/channels/src/components/admin_console/access_control/modals/cel_help/cel_help_modal.tsx

```diff
@@ -0,0 +1,85 @@
+                    id='admin.access_control.cel_help_modal.title'
+                    defaultMessage='Common Expression Language (CEL)'
+                />
+            )}
+            modalSubheaderText={(
+                <FormattedMessage
+                    id='admin.access_control.cel_help_modal.subheader'
+                    defaultMessage='With CEL you can define conditions to filter user attributes and control resource access.'
+                />
+            )}
+            compassDesign={true}
+            bodyPadding={false}
+            modalLocation='top'
+        >
+            <div className='cel-help-modal__content-container'>
+                <div className='cel-help-modal__content'>
+                    <Markdown
+                        message={'### Basic Syntax\nCEL expressions evaluate to boolean values (`true`/`false`) to determine if access should be granted.\n### Common Examples\n- To match a specific program:\n&ensp;`user.attributes.Program == "Delta"`\n- To match any of multiple teams:\n&ensp;`user.attributes.Team in ["Sales", "Engineering"]`\n- To match an email domain:\n&ensp;`user.attributes.Email.endsWith("example.com")`\n- To combine conditions (for this example with `OR` operator, altertanitvely use `&&` for `AND` operation):\n&ensp;`user.attrs.Program == "Alpha" || user.attrs.Team == "Operations"`\n### Supported Operators and functions\n- `==`, `!=`, `&&`, `||`, `in`, `contains()`, `startsWith()`, `endsWith()`'}
+                    />
+                </div>
+                <div className='cel-help-additional-info-modal__content'>
+                    <div className='cel-help-additional-info-modal__header'>
+                        <i className='icon icon-information-outline'/>
+                        <span className='cel-help-additional-info-modal__title'>
+                            <FormattedMessage
+                                id='admin.access_control.cel_help_modal.important_notes_title'
+                                defaultMessage='Important Notes'
+                            />
+                        </span>
+                    </div>
+                    <div className='cel-help-additional-info-modal__text'>
+                        <Markdown
+                            message={'- Operators like `<` or `>` are forbidden due to incorrect string comparison.\n- Only `user.attributes` are supported; any other variables are not supported yet.'}
+                        />
+                        <FormattedMessage
+                            id='admin.access_control.cel_help_modal.external_link'
+                            defaultMessage='For more information, visit <link>CEL Documentation</link>.'
+                            values={{
+                                link: (msg: React.ReactNode) => (
+                                    <ExternalLink
+                                        href='https://cel.dev/'
+                                        location='#'
```

Comment:
(Post-merge) This `location` is actually a value that's passed to the analytics code, not where we're navigating to. I don't think there's a strict naming convention for that, so we could change it to
```suggestion
                                        location='cel_help_modal'
```

Reply:
Oh thanks, I couldn't figure out actually and `#` seemed like a safe option.